### DISCUSSION TITLE
CANT OPEN DMG FILE COAPP

### DISCUSSION THREAD

**Discussion by levansm** [aclap-dev/video-downloadhelper#893] [Help and Q&A]
### Your question

Hello, 
I'm new here, and I'm already using Video Download Helper. I wanted to add CoApp but I have a problem with the dmg file. It doesn't start and it says I don't have a permission (weird because I ve checked the security and privacy setting and it settle on "anything". I forgot my pc is MacOS Macbook M2 Pro 14 inch). I tried different other setups (pkg and tar with install.sh, opened with DiskMounter as well. And nothing worked)  and checked the FAQ but can't fix my problem.

<img width="303" alt="Screenshot 2024-06-18 at 11 27 41" src="https://github.com/aclap-dev/video-downloadhelper/assets/156527153/9d2e5584-0a40-4323-834b-2cc737027ea8">

- Video Download Helper - Version 9.0.2.3 (stable), target mozilla, locale en-US.
- Companion App not installed

How should I fix it?? 

### Addon & CoApp version and details

Hello, 
I'm new here, and I'm already using Video Download Helper. I wanted to add CoApp but I have a problem with the dmg file. It doesn't start and it says I don't have a permission (weird because I ve checked the security and privacy setting and it settle on "anything". I forgot my pc is MacOS Macbook M2 Pro 14 inch). I tried different other setups (pkg and tar with install.sh, opened with DiskMounter as well. And nothing worked)  and checked the FAQ but can't fix my problem.

<img width="303" alt="Screenshot 2024-06-18 at 11 27 41" src="https://github.com/aclap-dev/video-downloadhelper/assets/156527153/9d2e5584-0a40-4323-834b-2cc737027ea8">

- Video Download Helper - Version 9.0.2.3 (stable), target mozilla, locale en-US.
- Companion App not installed

How should I fix it??

**Comment by paulrouget** [comment]
That's surprising. Are you comfortable with Terminal app? Can you do a md5sum of the pkg?

**Comment by levansm** [comment]
Thanks for your answer @paulrouget. Actually, it's first time to do a md5sum check. But I will google it or if you could provide me an example I will follow!

**Comment by levansm** [comment]
this is the hash after the command md5checksum -> 3591a8ea40c8c35a4e4f9c46694f3b92. I tried to open the file checksum on the git repo but I can't open it and check the value. Can you tell anything from the value I provided?
